tests/sqlite3/test-sqlite3.cpp

```diff
@@ -59,6 +59,15 @@ TEST_CASE("SQLite connection string", "[sqlite][connstring]")
 
 }
 
+// Test loading backend dynamically if possible (this symbol is defined by
+// the build system if shared libraries are used).
+#ifdef SOCI_DYNAMIC_BACKEND
+TEST_CASE("SQLite dynamic backend", "[sqlite][dynamic]")
+{
+    CHECK_NOTHROW(soci::session("sqlite3", "dbname=:memory: nocreate"));
```

Comment:
Might be nice if we could test dynamic loading for all tested backends (e.g. do it as part of the common tests).

Reply:
Yes, I admit I was too lazy to do it, but I've just done it now, thanks for poking me.